lib/core/elastic_connector_actions.rb

```diff
@@ -189,13 +190,15 @@ def create_job(connector_settings:)
           status: Connectors::SyncStatus::PENDING,
           created_at: Time.now,
           last_seen: Time.now,
+          trigger_method: connector_settings.sync_now? ? Connectors::JobTriggerMethod::ON_DEMAND : Connectors::JobTriggerMethod::SCHEDULED,
```

Comment:
Not a blocker however, if we stick to the plan of getting rid of all `business logic` from ElasticConnectorActions then I would move this line to `ConnectorSettings` or `ConnectorJob`.

Reply:
This makes sense. Can be done in a separate `elastic_connector_actions` refactor issue.